release_creation/bundle/bundle_os.py

```diff
@@ -0,0 +1,31 @@
+import platform
+
+from strenum import StrEnum
+
+
+class BundleOS(StrEnum):
+    LINUX = "linux"
+    MAC = "mac"
+    WINDOWS = "windows"
+
+    @classmethod
+    def get_local_os(cls) -> str:
+        local_sys = platform.system()
+        if local_sys == "Linux":
+            return BundleOS.LINUX
```

Comment:
Is this different than returning `cls("linux")`? I honestly don't know, I've just always done this using `cls` instead of returning the value explicitly.

Reply:
So I _think_  that what you're suggesting would return an instance of BundleOS while this just returns the enum value but I haven't played with it.